README.md

```diff
@@ -16,10 +16,8 @@ storages as well as simplified usage of different data encodings.
 Furthermore, it allows default values on config structs to be initialized
 from structs of the same type.
 
-## Install
+## Installing
 
-With a [correctly configured](https://golang.org/doc/install#testing) Go toolchain:
+To use `go-structconf` just add
+`github.com/anexia-it/go-structconf <version>` to your `go.mod` file.
```

Comment:
See equal discussion in https://github.com/anexia-it/go-structmapper/pull/3#discussion_r792762966

Reply:
As noted in the other discussion: explaining how Go modules work is out of the scope of our README file